src/ttsnake.c

```diff
@@ -333,28 +374,33 @@ void playgame (scene_t* scene)
 /* Process user input.
    This function runs in a separate thread. */
 
-void * userinput()
-{
-  int c;
-  while (1)
-    {
-      c = getchar();
-      
-      switch(c)
-	{
-	case 'q':
-	  kill (0, SIGINT);
-	  break;
-	default:
-	  break;
+void * userinput(){
+	int c;
+	while (1){
+		c = getchar();
+		switch(c){
+			case 'w':
+				snake.direction = up;
+				break;
+			case 'a':
+				snake.direction = left;
+				break;
+			case 's':
+				snake.direction = down;
+				break;
+			case 'd':
+				snake.direction = right;
+				break;
+			case 'q':
+			kill (0, SIGINT);
+			break;
+		default:
+			break;
+		}
```

Comment:
Inconsistent indentation and `default` case is not needed
```suggestion
		switch(c){
			case 'w':
				snake.direction = up;
				break;
			case 'a':
				snake.direction = left;
				break;
			case 's':
				snake.direction = down;
				break;
			case 'd':
				snake.direction = right;
				break;
			case 'q':
				kill (0, SIGINT);
				break;
		}
```